kernel/boot_info/src/lib.rs

```diff
@@ -38,7 +38,7 @@ pub trait ElfSection {
     /// Returns the section's starting virtual address.
     fn start(&self) -> VirtualAddress;
 
-    /// Returns the section's length.
+    /// Returns the section's length in memory.
```

Comment:
is this comment addition intended to contrast against the section's length in the ELF file? if so, let's say that too.